src/core/harpoon.ml

```diff
@@ -239,28 +239,27 @@ module Tactic = struct
          | Coverage.CovCtx _
            | Coverage.CovGoal (_, _, _) ->
             Misc.not_implemented "CovCtx impossible"
-         | Coverage.CovPatt (cG, p, tau) ->
-            let open Comp in
+         | Coverage.CovPatt (cG, pat, tau) ->
             let refine_ctx ctx = Whnf.cnormCtx (Whnf.normCtx ctx, ms) in
-            let cG = refine_ctx s.context.cG in
+            let cG = Coverage.compgctx_of_gctx cG in
             let cIH = refine_ctx s.context.cIH in
             dprintf
-              begin fun pr ->
-              pr.fmt "[harpoon-split] got pattern @[%a@]"
-                (P.fmt_ppr_cmp_pattern cD cG P.l0) p
+              begin
```

Comment:
I used the style
```
begin fun p ->
p.fmt "foo"
end
```
(almost) everywhere when I did the pretty-printer refactoring. I think we should keep with that style. I think it's nice actually that the indentation doesn't increase, since the `begin`-`end` stick out enough.

Reply:
@tsani OK, I will change one.

Reply:
@tsani OK, I will change one.

Reply:
@tsani  Resolved in 8e11337